sm_modified_cnn/create_dataset.sh

```diff
@@ -0,0 +1,16 @@
+mkdir data
```

Comment:
Please use `mkdir -p`. If this script is run multiple times `mkdir` would error since the directory already exists.